seirmo/_models.py

```diff
@@ -56,6 +87,51 @@ class SEIRModel(ForwardModel):
     def __init__(self):
         super(SEIRModel, self).__init__()
 
+        # Assign default values
+        self._output_names = ['S', 'E', 'I', 'R', 'Incidence']
+        self._parameter_names = [
+            'S0', 'E0', 'I0', 'R0', 'alpha', 'beta', 'gamma'
+        ]
+        # The default number of outputs is 5,
+        # i.e. S, E, I, R and Incidence
+        self._n_outputs = len(self._output_names)
+        # The default number of outputs is 7,
+        # i.e. 4 initial conditions and 3 parameters
+        self._n_parameters = len(self._parameter_names)
+
+        self._output_indices = np.arange(self._n_outputs)
+
+    def n_outputs(self):
+        # Return the number of outputs
+        return self._n_outputs
+
+    def n_parameters(self):
+        # Return the number of parameters
+        return self._n_parameters
+
+    def output_names(self):
+        # Return the (selected) output names
+        return self._output_names
+
+    def parameter_names(self):
+        # Return the parameter names
+        return self._parameter_names
+
+    def set_outputs(self, outputs):
+        # Check existence of outputs
+        for output in outputs:
+            if output not in self._output_names:
+                raise ValueError('The output names specified must be in correct forms') # noqa
```

Comment:
Here it's better to break the line instead of using noqa, i.e.
```
for output in outputs:
     if output not in self._output_names:
          raise ValueError(
              'The output names specified must be in correct forms')